code/config/lib/Loot/Config/CLI.hs

```diff
@@ -64,20 +48,60 @@ type Config =
 And here is a valid parser for this config:
 
 @
-cfgParser :: OptParser Config
+cfgParser :: OptModParser Config
 cfgParser =
-    #heh  .:: switch (long "heh") .<>
+    #heh  .:: switch (long "heh") <.>
     #taks .:<
-       (#chto  .:: strOption (long "chto") .<>
+       (#chto  .:: strOption (long "chto") <.>
         #u_nas .:: switch (long "u_nas"))
 @
 
 Note that options "mda" and "taks.tut" are omitted
 from parser's definition. This means that these options
 will not be included in the partial config produced by the parser.
 -}
-(.<>)
-    :: Semigroup (ConfigRec 'Partial is)
-    => OptParser is -> OptParser is -> OptParser is
-p .<> q = (<>) <$> p <*> q
-infixr 5 .<>
+(<.>) :: ModParser a -> ModParser a -> ModParser a
+p <.> q = (.) <$> p <*> q
+infixl 5 <.>
```

Comment:
This is a really minor point, but this operator clashes with more general operator of [Apply](http://hackage.haskell.org/package/semigroupoids-5.3.1/docs/Data-Functor-Apply.html#t:Apply) which may be useful one day (we already have one potential instance of `Apply` in disciplina).